shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/loader/dialect/MySQLSchemaMetaDataLoader.java

```diff
@@ -125,6 +125,9 @@ private void appendDataTypes(final Map<String, Integer> dataTypes) {
         dataTypes.putIfAbsent("JSON", Types.LONGVARCHAR);
         dataTypes.putIfAbsent("GEOMETRY", Types.BINARY);
         dataTypes.putIfAbsent("YEAR", Types.DATE);
+        dataTypes.putIfAbsent("MULTIPOINT", -2);
```

Comment:
@mabaiwan  Hi, Thank you for your pull reqeust. Can you add unit test for them?

Reply:
of course.